src/App.jsx

```diff
@@ -0,0 +1,46 @@
+import { useState } from 'react';
+import TodoInput from './TodoInput';
+import TodoList from './TodoList';
```

Comment:
```suggestion
import { useState } from 'react';

import TodoInput from './TodoInput';
import TodoList from './TodoList';
```

의미가 구분될 때에는 빈 줄을 넣어서 구분해주세요.
빈 줄에도 하나하나에도 의미가 담겨있다고 생각하시고 코드를 작성하시는 연습을 하시면 좋아요.

Reply:
그냥 넘겼던 부분들은 다 리뷰의 대상이 되네요..
그만큼 생각 없이 작성한 부분이 많았던 것 같습니다.
수정하겠습니다!